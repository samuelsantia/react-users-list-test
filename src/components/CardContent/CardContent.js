import styled from 'styled-components/macro';

const CardContent = styled.div`
  padding: var(--card-padding, 1rem);
`;

export default CardContent;
