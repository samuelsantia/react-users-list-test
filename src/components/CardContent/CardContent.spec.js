import React from 'react';
import { render } from '@testing-library/react';
import 'jest-styled-components';

import CardContent from './CardContent';

describe('<CardContent />', () => {
  it('should render', () => {
    const { container } = render(<CardContent>Test CardContent</CardContent>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
