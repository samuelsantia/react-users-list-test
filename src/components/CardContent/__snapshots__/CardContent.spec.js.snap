// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardContent /> should render 1`] = `
.c0 {
  padding: var(--scale-rhythm,1rem) var(--card-padding,1rem) var(--card-padding,1rem);
}

.c0:first-child {
  padding-top: var(--card-padding);
}

<div
  class="c0"
>
  Test CardContent
</div>
`;
